src/components/Filter/Filter.jsx

```diff
@@ -0,0 +1,83 @@
+import '../Filter/filter.css'
+
+function Filter() {
```

Comment:
Suggestion: I think this should be named as Sidebar, because filter is a functionality